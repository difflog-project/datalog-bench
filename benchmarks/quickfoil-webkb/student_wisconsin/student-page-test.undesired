1030
3571
4165
4032
3510
3657
3319
3472
3414
3784
3470
3767
3607
3574
3233
3814
1072
4075
3675
3677
3397
1009
1034
3760
3367
4114
3403
3631
3445
3475
3973
4105
3886
3284
3230
3799
3927
3898
3830
4056
3238
3774
4141
3315
3555
3822
561
3659
3716
3763
4082
721
3863
4145
4057
3343
3469
3897
3263
777
4153
3234
3484
3789
3669
1012
3723
4139
3991
3746
3770
4077
3366
1085
559
3964
3836
3448
754
3371
3329
3907
733
4112
3752
3697
4044
3427
773
3688
3823
4078
1047
4023
3896
3952
3275
1078
3768
3455
1084
3895
3837
3934
1005
3496
3672
3494
4070
3648
3753
3732
3516
3637
3307
3929
3628
4147
3786
3596
3714
775
740
3586
3959
3832
1059
3333
3262
3726
3296
4101
3855
1033
3963
1013
1082
1041
4092
3228
4008
4010
3985
3491
3303
3314
3664
3936
4054
3916
3652
781
4055
3310
3892
3270
3745
3416
3486
3739
3917
3272
3967
3553
4138
3490
1004
3910
4030
1026
3806
601
4016
1051
1039
3339
4164
3944
3984
4021
1066
3604
3354
1036
3570
3829
730
3981
744
3725
3619
3718
3464
776
3526
3791
3925
4143
4127
1080
1050
3943
3600
3389
3988
3743
3258
602
3549
1054
3683
3986
3287
3803
3990
3819
3429
3874
1020
4123
3670
764
3349
771
1043
732
3536
1061
4066
3567
3497
4025
3499
3646
3942
4161
4160
3900
3704
3580
3853
1067
3871
3353
3982
3518
3873
747
3658
3667
4084
3583
3541
1037
3833
3313
3559
738
3681
3458
3779
3413
4096
3309
3651
3337
4067
3647
783
3362
3249
765
3535
3346
3946
3462
3643
3566
4087
3449
3738
3513
1040
4026
3468
3393
3828
1044
3556
3477
737
766
3320
3966
3792
3722
3383
3282
3375
3776
1046
3968
3245
3454
4129
4058
3227
3788
3737
742
3762
3634
3850
564
3271
3922
3848
3655
3678
3695
4110
4012
3467
4156
3919
3693
3291
4045
3587
3735
3520
1075
3804
3504
1015
3838
3452
3412
3622
3633
3771
3340
3854
3912
1042
3709
3588
3769
4150
3713
3457
3772
3591
3483
565
3719
4111
3915
3808
3849
3401
3500
3225
3286
3794
3905
4115
3358
3298
3750
1079
3933
3441
3428
1083
3539
3610
3818
1025
3476
772
3406
4142
1064
3289
1076
4061
3426
3594
717
3603
3519
3727
3239
3639
4063
3785
3336
3511
4053
3835
3501
4060
4013
3865
3417
769
3359
3856
3640
725
1073
4144
4148
3345
1069
3423
3958
3674
3408
1010
3293
3266
3980
4073
726
4015
4117
3744
3351
3300
3625
3877
3888
3860
4038
558
3632
4080
3679
3317
3906
3885
3757
4130
3498
4158
3721
3415
4154
3370
4134
4014
3887
3301
1049
3685
746
3288
3256
3748
736
3715
4065
3845
3369
3705
3252
3243
755
4135
3777
778
3384
3407
3931
3444
3661
3805
743
1052
3949
3831
3577
3253
3560
4001
3395
3700
1019
3385
3790
4099
3843
3410
3306
3875
4083
1048
3305
3997
3834
3558
3348
3437
4103
4006
3350
3495
3505
3554
4149
3654
3264
3813
4007
3503
3778
3399
566
3926
3424
3502
3889
1027
3765
3474
3783
3724
3903
3882
3759
3800
770
3278
3450
3821
3481
3290
3589
3613
4136
4124
718
3509
3341
3945
3422
752
3939
3696
3820
3299
3283
3974
3515
3236
3542
3627
3564
3328
3402
3656
4094
3579
3827
3712
759
3728
3965
3807
3530
3431
3251
4027
3998
3529
3338
3590
3443
3621
3379
4031
3876
4003
3492
3242
1053
4166
3970
731
3355
3388
3391
3527
3440
3867
3405
3565
3438
3247
3825
3279
1031
3323
4046
3235
3826
4047
3368
3699
3533
3432
3222
719
3582
3525
3995
1007
749
562
4104
4126
3246
3387
734
3330
3913
3953
3878
3962
3992
3221
3593
727
745
3954
3514
3425
3844
3666
3707
4051
3869
3240
3710
3463
1022
3522
3308
1087
1011
3597
3280
1057
3575
3398
3364
3798
3937
3569
784
1081
3720
3618
723
3456
3342
3901
3969
4120
4028
3528
3335
3923
4098
3331
3232
1038
4086
4157
3400
3363
761
1008
4050
3276
3645
3326
3731
3324
3517
4052
1056
3373
3812
3466
1088
3436
3531
3987
3851
3523
4131
3689
3478
741
4022
3858
3928
3223
4064
1070
3642
1065
3764
3971
1058
3635
4106
3795
3740
774
3841
3660
3864
3975
3507
3606
3692
4116
716
4081
3376
3680
3924
3950
3879
4039
3846
3599
3904
722
3381
3638
3550
3626
3485
1074
3698
603
3961
3248
748
4002
3999
3435
3269
3390
3840
3956
3880
3479
3267
3273
3277
4017
1062
4000
3465
3322
3996
3609
4004
1006
3546
1018
3781
3703
4113
3957
3793
3508
3730
4091
1021
3782
4128
3562
3756
3749
3899
3948
3386
3506
3581
3960
1063
3747
3365
3404
4033
3918
1060
3920
3951
3545
3537
3624
4049
4085
763
3780
4036
3557
4162
3292
3281
3451
3810
3751
3244
3255
3891
3265
3989
3890
3708
3304
3229
3862
750
3994
3847
3894
4018
3344
3334
3360
3254
3471
3576
4088
3662
4151
3259
3442
3711
756
3312
3940
3394
3736
3671
3938
4074
3493
3473
3706
3601
3325
3914
4100
3839
3983
4079
3742
3809
3932
3487
3870
3611
3446
729
4093
4102
3614
3686
3347
4140
3881
4029
3573
3257
3668
3374
3883
4121
3327
3761
3842
3378
3653
1016
3717
3568
3551
3673
3694
3993
3602
3382
3815
3524
3552
3572
3690
4132
1055
3453
762
724
3377
3644
4133
780
3797
563
1014
3396
715
3649
4076
3857
4062
3665
3318
3921
3361
3796
3629
3311
1032
3641
4043
3544
4005
3608
3908
1045
3297
4090
4163
4118
3418
1024
785
3321
4146
4020
3352
3260
4072
1068
3630
3687
3605
4024
3754
3419
3868
3811
1028
3532
735
1077
3741
768
3224
753
1086
3941
3302
3578
3861
4152
3702
4109
4042
3285
4071
3859
4069
3617
3930
3540
3612
4089
3547
4037
728
720
3420
782
3733
3816
3978
3231
3872
3977
3972
3911
3447
3250
3316
751
4035
3598
3636
3563
3489
3461
3592
4011
3801
3434
3691
4097
4059
3421
3620
3561
3935
3701
3548
4009
3585
3775
3682
4041
3866
3595
3372
3852
3684
3650
3488
1071
4095
4048
4119
3616
4034
1023
4137
4068
3380
3902
3482
4019
3802
3433
3241
3332
4107
3734
3459
3615
3979
3773
739
767
3676
3409
3766
3357
3356
3392
4125
3884
3817
3663
3295
3237
4040
3480
3909
3294
1035
3521
3584
3787
3411
3755
3955
4159
3460
3534
560
4155
4122
779
3268
3538
3430
3261
3512
3947
3824
3893
1029
3226
3439
3729
3623
3543
3274
1017
3976
3758
4108
