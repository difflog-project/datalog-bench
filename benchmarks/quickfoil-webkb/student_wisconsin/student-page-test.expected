129
145
68
114
95
142
133
118
7
116
18
77
41
50
143
15
46
13
99
104
54
4
119
78
35
1
70
27
42
19
91
28
93
112
152
81
94
144
63
153
52
83
86
45
8
150
92
149
101
37
128
134
56
79
9
6
0
69
141
3
110
23
120
62
58
10
44
87
130
100
30
89
123
151
5
21
14
84
65
127
20
139
72
49
125
73
29
22
85
140
96
12
76
34
132
80
124
57
31
60
59
55
66
40
38
126
61
103
88
108
97
138
136
148
75
32
137
122
39
36
131
74
135
16
64
2
71
90
155
17
115
121
154
43
107
26
146
111
106
11
147
33
102
109
53
82
51
105
25
48
113
24
47
98
67
117
